README.md

```diff
@@ -176,7 +185,7 @@ before submitting your pull request.
 
 ## Bro-tie
 For the brofessional. Want to use Brototype.js but it's too bro for your work
-environment? Just give it the [Bro-tie](http://brotie.jdauriemma.com) treatment
+environment? Just give it the [Bro-tie](http://pepperthecat.github.io/brotie/) treatment
```

Comment:
Right? That's what I'm talking about. It's like it rebased to an older commit or something ⊙_⚆